omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -44,103 +47,167 @@
 @Deprecated
 @Controller
 public class OwaRestController {
-	
+
 	private static final Log log = LogFactory.getLog(OwaRestController.class);
-	
+
 	// -------------------------------------------------------------------------
 	// Dependencies
 	// -------------------------------------------------------------------------
 	@Autowired
 	private AppManager appManager;
-	
+
 	@Autowired
 	private MessageSourceService messageSourceService;
-	
+
 	// -------------------------------------------------------------------------
 	// REST implementation
 	// -------------------------------------------------------------------------
 	@RequestMapping(value = "/rest/owa/applist", method = RequestMethod.GET)
-		@ResponseBody
-		public List<App> getAppList() {
-			List<App> appList = new ArrayList<>();
-	        if(Context.hasPrivilege("Manage OWA")){
```

Comment:
Did you actually make any changes to this file or just formatting. If formatting, can you create a separate pull request for formatting? Such that this pull request has only the actual changes required by the ticket.